index.ts

```diff
@@ -366,7 +366,8 @@ export class OAuth2AuthCodePkceClient {
             + `code=${encodeURIComponent(authorizationCode || '')}&`
             + `redirect_uri=${encodeURIComponent(redirectUrl)}&`
             + `client_id=${encodeURIComponent(clientId)}&`
-            + `code_verifier=${codeVerifier}`;
+            + `code_verifier=${codeVerifier}&`
+            + `${objectToQueryString(oneTimeParams)}`;
```

Comment:
The string template with the variable in it is superfluous. Simply append the result of the function.